### PR TITLE
✨ Feature: Ground Rule

### LINKED ISSUE #1: 창섭#1 merge
## 🛠️ Planned development features
- 팀 내부 규칙 정하기 

## 📝 check-lists
- [x] txt file인지 확인하기

### PR DESCRIPTION
**Pull request by kurtyoon**
## Issue
You need to link it to the index specifying which issue you addressed.

- Resolves : #1 
 
## What To Do
You should present a list of what you've done in the corresponding pull request.

- ground rule 작성

### CODE REVIEW

File: Rule.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1. 카카오톡 및 디스코드 읽씹 및 안읽씹 금지
+2. 정기미팅 지각 시 벌금 5000원 및 결석 시 벌금 10000원
```
Comment (jjuuuunnii) [inline Rule.txt:2]: 우앗 이거 5000원 빡시다

**Review by jjuuuunnii — APPROVED** [review]
빡센 룰 확인이요,,,

File: Rule.txt (hunk 1 above)
Comment (shimseohyun) [reply Rule.txt:2]: 벌금은 어디에 쓰이나요?

**Review by shimseohyun — APPROVED** [review]

File: Rule.txt (hunk 1 above)
Comment (jjuuuunnii) [reply Rule.txt:2]: 회식비,,?

**Pull request merged**